Emby.Server.Implementations/SocketSharp/WebSocketSharpRequest.cs

```diff
@@ -326,7 +326,7 @@ public static string LeftPart(string strVal, char needle)
                 return null;
             }
 
-            var pos = strVal.IndexOf(needle, StringComparison.Ordinal);
+            var pos = strVal.IndexOf(needle.ToString(), StringComparison.Ordinal);
```

Comment:
Why this change?

Reply:
Apologies for the late reply, dotnet standard doesn't seem to have the implementation. The code used to be in Jellyfin.Server which is a dotnet core project which does have this overload. The code is now in Server.Implementations, a dotnet standard project, which is why I had to make this change.